man/labgrid-client.rst

```diff
@@ -75,6 +78,12 @@ This variable can be used to specify a state which the device transitions into
 before executing a command. Requires a configuration file and a Strategy
 specified for the device.
 
+LG_INITIAL_STATE
+~~~~~~~~~~~~~~~~
+This variable can be used to specify an initial state the device is known to
+be in. This is useful during development. The Strategy used must implement the
+``force()`` method. Requires a desired state.
```

Comment:
Here as well. Please also use semantic newlines.

Reply:
We didn't follow that rule in the man page, but I'll start now.